File: ElectionGuard.BallotTracker.Functions/Models/Tally.cs
namespace ElectionGuard.BallotTracker.Functions.Models
{
    public class Tally
    {
        public string SelectionId { get; set; }

        public string Name { get; set; }

        public string Description { get; set; }

        public string Party { get; set; }

        public int VoteCount { get; set; }
    }
}

File: src/models/tally.ts
/**
 * The complete tallied results of an election.
 *
 * Note that this does not match directly with any core ElectionGuard models,
 * but is a transformed combination of the Election description and Tally results.
 */
export interface ElectionResultsSummary {
    election_id: string;
    election_results: ElectionResults;
}

export interface ElectionResults {
    [contest_id: string]: ContestResults;
}

export interface ContestResults {
    [selection_id: string]: number;
}
